apollo-federation-types/src/javascript/mod.rs

```diff
@@ -69,3 +77,52 @@ pub struct GraphQLError {
 pub struct GraphQLErrorExtensions {
     pub code: String,
 }
+
+impl TryFrom<SubgraphDefinition> for Subgraph<Initial> {
+    type Error = SubgraphError;
+
+    fn try_from(value: SubgraphDefinition) -> Result<Self, Self::Error> {
+        Subgraph::parse(value.name.as_str(), value.url.as_str(), value.sdl.as_str())
+    }
+}
+
+impl TryFrom<SubgraphDefinition> for Subgraph<Upgraded> {
```

Comment:
instead of try from trait move it to a separate method to make it explicit that we are ignoring validation